apps/happy-blocks/block-library/support-content-footer/style.scss

```diff
@@ -0,0 +1,182 @@
+@import "@automattic/calypso-color-schemes";
+
+$breakpoint-mobile: 782px; //Mobile size.
+$breakpoint-tablet: 1224px; //Large tablet size.
+$breakpoint-desktop: 1440px; //Desktop size.
+$search-icon: 'data:image/svg+xml,%3Csvg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none"%3E%3Cpath d="M9.16667 15.8333C12.8486 15.8333 15.8333 12.8486 15.8333 9.16667C15.8333 5.48477 12.8486 2.5 9.16667 2.5C5.48477 2.5 2.5 5.48477 2.5 9.16667C2.5 12.8486 5.48477 15.8333 9.16667 15.8333Z" stroke="%238C8F94" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/%3E%3Cpath d="M17.5 17.5L13.875 13.875" stroke="%238C8F94" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/%3E%3C/svg%3E';
+$recoleta-font-family: "Recoleta", sans-serif;
+$system-font-family: -apple-system, BlinkMacSystemFont, "Noto Sans", "Segoe UI", "Roboto",
+	"Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", sans-serif;
+
+.happy-blocks-support-content-footer {
+	max-width: none !important;
+	align-items: normal !important;
+	box-sizing: border-box;
+	padding-bottom: 20px;
+	padding-top: 20px;
+
+	p {
+		font-size: 1rem;
+		line-height: 23px;
+	}
+
+	h4 {
+		font-size: 1.75rem;
+		margin-bottom: 0;
+	}
+
+	>* {
+		margin-left: auto !important;
+		margin-right: auto !important;
+		max-width: var(--wp--style--global--wide-size);
+	}
+
+	.support-content-cta {
+		border: 1px solid var(--studio-gray-5);
+		display: flex;
+		margin-bottom: 2rem;
+		font-size: 1.25rem;
+
+		.support-content-cta-left {
+			flex: 1;
+			padding: 20px;
+			padding-top: 40px;
+
+			h4 {
+				line-height: 52px;
+			}
+		}
+
+		.support-content-cta-right {
+			flex: 1;
+
+			figure {
+				margin-bottom: 0;
+				height: 100%;
+
+				img {
+					width: 100%;
+					height: 100%;
```

Comment:
This ruins the aspect ratio of the picture. One of them should be 100%, I think height.